content/diversity-target.html

```diff
@@ -1,6 +1,8 @@
 type: page
 title: Diversity Target
 slug: diversity-target
+callout_big_1: We're measurable
```

Comment:
I'm not wild about this as a title, but I can't think of anything better beyong "Our target for diversity".